### PR TITLE
Add more info to the documentation

### PR DESCRIPTION
**Pull request by rowild**
- General git workflow
- Infos on how to set up mkdocs and its dependencies to build and serve locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,120 @@
...
+Please refer to sources on the internet for explanations on how to install python 3 on
+your operating system. One such resource would be [Real Python: Installing Python](https://realpython.com/installing-python/).
+
+### Install mkdocs and required dependencies
+
+`aimeos-docs` uses [mkdocs](https://www.mkdocs.org/) to create static html files from
+markdown files. Install `mkdocs` and the dependencies required by `aimeos-docs` with `pip`:
 
 ```bash
 pip install mkdocs material
```
Comment (aimeos) [inline README.md:108]: There's a mistake in our existing docs. It must be:
```
pip install mkdocs mkdocs-material
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,120 @@
...
 ```
 
-Afterwards, run `mkdocs build` to generate the documentation. If you want to check the generated documentation in your browser, you can use `mkdocs serve` instead. Afterwards, you can have a look at the pages at `http://127.0.0.1:8000/`.
\ No newline at end of file
+#### Start the local server
```
Comment (aimeos) [inline README.md:111]: This should go one level up, so could you use `###`?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,120 @@
...
+
+Run `mkdocs serve` to start a local server on `http://127.0.0.1:8000`. Now, whenever
+you make changes to any file, the server will automatically reload and display the new
+version.
+
+#### Build the docs
```
Comment (aimeos) [inline README.md:117]: Same here

**Comment by rowild** [comment]
Hmmm... The idea was to provide a "git chapter" (required for commits) and a "mkdocs chapter" (optinal). Hence I would argue that these 2 subheadings belong to the mkdocs section and should therefore be subordinated to "Install mkdocs and required dependencies".

Why would you want to pull those headings up a level?

**Comment by rowild** [comment]
Oh, I'm sorry - there is anther level for that already... I make the changes. Just a moment...

File: README.md (hunk 3 above)
Comment (rowild) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (rowild) [reply README.md:108]: Done.

Comment (rowild) [reply README.md]: Done.

**Pull request merged**